app/components/product/edit.hbs

```diff
@@ -36,6 +38,30 @@
                 @value={{@model.name}}
                 class="mt-1 focus:ring-red-200 focus:border-red-200 block w-full border-gray-300 rounded-md sm:text-sm" />
             </div>
+            <div class="md:col-span-4">
+              <div class="flex">
+                <label for="product-alias" class="block text-sm font-medium text-gray-700">
+                  Aliasen
+                </label>
+                <div>
+                  <Rlv::Button
+                    @icon="arrow-down-s-line"
+                    @skin="secondary-link"
+                    @size="small"
+                    class="mt-1 ml-2"
+                    {{on "click" this.toggleAliasForm}} />
```

Comment:
Making them collapsible is indeed a bit overkill. They probably want to see them all at once.